app/google-services.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
@mohitmanuja Please do not add these keys to the public repo.
If you plan on adding them, create a dummy project on firebase and then add the son from that over here.